src/lib/edu-api.js

```diff
@@ -0,0 +1,15 @@
+import superagent from 'superagent';
+import superagentJsonapify from 'superagent-jsonapify';
+import apiClient from 'panoptes-client/lib/api-client';
+import { config } from './config';
+
+superagentJsonapify(superagent);
+
+export function get(endpoint) {
```

Comment:
We should add another argument to this helper method to take query params and build those into the request URL to later support things like filters.